docs/tutorials/hello-minikube/hello-minikube.md

```diff
@@ -0,0 +1,305 @@
+
+#### Dockerfile
+
+{% include code.html language="conf" file="Dockerfile" ghlink="/docs/tutorials/hello-minikube/Dockerfile" %}
+
+This "recipe" for the Docker image will start from the official Node.js LTS image found on the Docker registry, 
+expose port 8080, copy our `server.js` file to the image and start the Node server.
+
+Since this tutorial uses Minikube, instead of pushing Docker images to a
+registry, you can simply build the image using the same Docker host as
+the Minikube VM, so that the images are automatically present. To do so, make
+sure you are using the Minikube docker daemon
+
+```shell
+eval $(minikube docker-env)
+```
+
+If you no longer with to use the Minikube host, you can undo this change by
+running the same command with the `-u` flag.
+
+Now build an image of your container by running `docker build`, tagging the 
+image your DockerHub username. Make sure you build the image using the 
+Minikube daemon.
+
+```shell
+docker build -t hello-node:v1 .
+```
+
+Now the Minikube VM can run the image you built.
+
+## Create your pod
+
+A Kubernetes **[pod](/docs/user-guide/pods/)** is a group of containers, tied 
+together for the purposes of administration and networking. It can contain a 
+single container or multiple.
+
+Create a Pod with the `kubectl run` command:
+
+```shell
+kubectl run hello-node --image=hello-node:v1 --port=8080
+```
+
+As shown in the output, the `kubectl run` created a **[Deployment](/docs/user-guide/deployments/)** object.  
+Deployments are the recommended way for managing creation and scaling of pods.  
+In this example, a new deployment manages a single pod replica 
+running the *hello-node:v1* image.
+
+To view the Deployment we just created run:
+
+```shell
+kubectl get deployments
+
+NAME         DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+hello-node   1         1         1            1           3m
+```
+
+To view the Pod created by the deployment run:
+
+```shell
+kubectl get pods
+
+NAME                         READY     STATUS    RESTARTS   AGE
+hello-node-714049816-ztzrb   1/1       Running   0          6m
+```
+
+To view the stdout / stderr from a Pod run (probably empty currently):
+
+```shell
+kubectl logs <POD-NAME>
+```
+
+To view cluster events run:
+
+```shell
+kubectl get events
+```
+
+To view the kubectl configuration run:
+
+```shell
+kubectl config view
+```
+
+Full documentation for kubectl commands is available **[here](/docs/user-guide/kubectl-overview/)**:
+
+## View your service
+
+By default, the pod is only accessible by its internal IP within the Kubernetes cluster. 
+In order to make the `hello-node` container accessible from outside the 
+Kubernetes virtual network, you have to expose the Pod as a 
+Kubernetes **[Service](/docs/user-guide/services/)**.
+
+From our Development machine we can expose the pod to the public internet 
+using the `kubectl expose` command.  
+
+```shell
+kubectl expose deployment hello-node --type=LoadBalancer         
```

Comment:
I agree it's confusing, because on the one hand "LoadBalancers" don't actually exist in Minikube. On the other hand, if you don't use it, `minikube service` doesn't work. Perhaps we should file an issue on Minikube to make Minikube service work regardless, but not sure what else we can do here. It makes a little bit of sense to me, because in both cases you are exposing a Service outside of the cluster.